website/docs/kubernetes/kind.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 1
+title: Kind support
+description: Kind is one way to get Kubernetes running on your workstation.
+keywords: [podman desktop, podman, containers, migrating, kubernetes, kind]
+tags: [migrating-to-kubernetes, kind]
+---
+
+# Kind
+
+Kind is a command line tool that can create Kubernetes clusters on your favorite container engine.
+
+It has experimental support for Podman. However, Kind has specific requirements that needs configuration tuning.
+
+## Windows and WSL
+
+Due to incompatibilities between WSL and systemd, Kind does not work with the rootless mode. So if you want to use Kind
+with your Podman machine, you need to configure rootful mode.
+
+### Existing Podman machine
+
+When you create a Podman machine, two system connections will be created. One of them is the rootless one and the other
+is the rootful. As kind will use the default Podman connection, you need to make sure it is the rootful.
+
+Run the following command:
+
+```shell
+podman system connection ls
+```
+
+You should see a similar output:
+```
+Name                         URI                                                          Identity                                   Default
+podman-machine-default       ssh://user@localhost:54133/run/user/1000/podman/podman.sock  C:\Users\Jeff\.ssh\podman-machine-default  true
+podman-machine-default-root  ssh://root@localhost:54133/run/podman/podman.sock            C:\Users\Jeff\.ssh\podman-machine-default  false
+```
+
+The rootful connection is suffixed by `-root`. If it is not the default one, you must issue the following command:
+
+```shell
+podman system connection default podman-machine-default-root
+```
```

Comment:
(scanability)
(scanability)
(logical order)
(nested blocks)
(minimalism)

```suggestion
#### Procedure

* Set the default Podman system connection: 

  ```shell
  podman system connection default podman-machine-default-root
  ```

#### Verification
 
* List the Podman system connections:

  ```shell
  podman system connection ls
  ```

  You should see a similar output:

  ```
  Name                         URI                                                            Identity                                   Default
  podman-machine-default       ssh://user@localhost:54133/run/user/1000/podman/podman.sock  C:\Users\Jeff\.ssh\podman-machine-default  true
  podman-machine-default-root  ssh://root@localhost:54133/run/podman/podman.sock            C:\Users\Jeff\.ssh\podman-machine-default  false
  ```

  The rootful connection is suffixed by `-root`.

Reply:
The list of podman connections was to handle case where the podman machine is podman-machine-default, I'm afraid removing it will confuse users

Reply:
Indeed, I'll update the proposal

Reply:
Better?